api/nakadi-event-bus-api.yaml

```diff
@@ -606,22 +488,388 @@ paths:
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporarily) unavailable
+          schema:
+            $ref: '#/definitions/Problem'
+
+  /subscriptions/{subscription_id}/cursors:
+    get:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Exposes the currently committed offsets of a subscription.
+      parameters:
+        - $ref: '#/parameters/SubscriptionId'
+      responses:
+        '200':
+          description: Ok
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '403':
+          description: Client is not authorized to perform this operation
+          schema:
+            $ref: '#/definitions/Problem'
+        '404':
+          description: Subscription not found
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Service (temporarily) unavailable
+          schema:
+            $ref: '#/definitions/Problem'
+    put:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: |
+        Endpoint for committing offsets of the subscription.
+        The behavior of commit is specific to `commit_mode` and `commit_semantics` parameters of the session
+        that makes a commit. Please see details in subscription streaming endpoint specification.
+      parameters:
+        - name: subscription_id
+          in: path
+          type: string
+          description: Id of subscription
+          required: true
+        - name: X-Nakadi-SessionId
           in: header
+          type: string
+          description: |
+            Id of a session (client) which makes a commit. It is not possible to make a commit from a terminated or
+            none-existing session. Also the session can't commit something which was not sent to it.
+          required: true
+        - name: cursors
+          in: body
+          description: Cursors with offsets to commit.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Cursor'
+          required: true
+      responses:
+        '200':
+          description: Offsets were committed
+        '204':
+          description: |
+            No Content. At least one cursor which was tried to be committed is older or equal to already committed one.
```

Comment:
as a client, how should i deal with partial failure? it would be helpful to know which cursors were committed successfully, and which were not (and why).